api/src/services/fields.ts

```diff
@@ -535,6 +545,14 @@ export class FieldsService {
 								if (!hookAdjustedField.schema) return;
 								this.addColumnToTable(table, collection, field, existingColumn);
 							});
+
+							await this.trackFieldSchemaChange(
```

Comment:
This will always throw in the upsert case, as the record doesn't exist until L575 (when applicable).